exonum-java-binding-core/src/test/java/com/exonum/binding/proxy/NativeHandleTest.java

```diff
@@ -23,53 +23,53 @@
 
 import org.junit.jupiter.api.Test;
 
-
 class NativeHandleTest {
 
   private NativeHandle nativeHandle;
 
+  private static long HANDLE = 0x11L;
+
+  private static final String NATIVE_HANDLE = "11";
```

Comment:
I can't say it is better, it is not the same constant that you reuse in code because it *must* have the same value. It just _happens_ that tests use the same value, therefore, extracting these constants does not make them clearer, on contrary — it makes them harder to understand, because you can't immediately see that the second is another representation of the first.

On top of that, the naming does not help either.